src/myapp.h

```diff
@@ -47,6 +47,22 @@ struct MyApp : wxApp {
 
             // wxWidgets should really be doing this itself, but it doesn't (or expects you to
             // want to use a manifest), so we have to call it ourselves.
+
+            // Attempt to load the different HiDPI API functions provided in User32 and Shcore
+
+            HMODULE user32 = LoadLibraryA("User32.dll");
+            typedef BOOL (WINAPI * SETPROCESSDPIAWARE_T)(void);
+            typedef BOOL (WINAPI * SETPROCESSDPIAWARENESSCONTEXT_T)(DPI_AWARENESS_CONTEXT);
+            SETPROCESSDPIAWARE_T SetProcessDPIAware = NULL;
+            SETPROCESSDPIAWARENESSCONTEXT_T SetProcessDpiAwarenessContext = NULL;
+            if (user32) {
+                SetProcessDPIAware =
+                    (SETPROCESSDPIAWARE_T)GetProcAddress(user32, "SetProcessDPIAware");
+                SetProcessDpiAwarenessContext =
+                    (SETPROCESSDPIAWARENESSCONTEXT_T)GetProcAddress(user32, "SetProcessDpiAwarenessContext");
+            }
+
+            HMODULE shcore = LoadLibraryA("Shcore.dll");

```

Comment:
Make this code a bit more organized: put both LoadLibrary calls next to eachother, put all 3 variables next to eachother, etc

Reply:
Thanks for your review! Does the revision fit better now?